### PR TITLE
[CONFLUENCE] get rid of 0.75 effectslowdown

### PR DESCRIPTION
**Pull request by phil65**
This PR aims to get rid of the effectmultiplier used by Confluence. That one is a real pain and we should get rid of it sooner than later because it also applies to python scripts which provide their own XMLs for example.  Also, a lot of other skinners base their work on confluence and a lot of them never notice that they don´t work with the "real" time values and never change it back. In fact we should "disallow" using any other value than 1.0 for repo skins (can still get used for debug purposes of course).
Atm I am scaling:
- attributes "time", "delay", "repeat" for elements "animation", "effect" and "scrolltime"
- "autoscroll" element

I´m using my XML tool to do this, so adding other values is easy. Please let me know if I got everything. :)
Downside is that we do not have round values anymore. If wanted then I could round it to 100ms steps, 10ms steps whatever. Up to you guys.
@ronie, @BigNoid for review, please.

### CODE REVIEW

**Comment by phil65** [comment]
oops, i scaled in the wrong direction :) will fix it. I´m more interested for now in whether I got all values which needs to get converted though.

File: addons/skin.confluence/720p/IncludesHomeRecentlyAdded.xml [hunk 1]
```diff
@@ -140,7 +140,7 @@
 					<content>
 						<item>
 							<label>$INFO[Window.Property(LatestMovie.1.Title)]</label>
-							<label2/>
+							<label2 />
```
Comment (BigNoid) [inline addons/skin.confluence/720p/IncludesHomeRecentlyAdded.xml]: This must be an error in the skinning tool :)

Comment (phil65) [inline addons/skin.confluence/720p/IncludesHomeRecentlyAdded.xml]: Well it´s just the way the XMLWriter formats the stuff. I am sure it´s according to specifications.

Comment (phil65) [inline addons/skin.confluence/720p/IncludesHomeRecentlyAdded.xml]: ...in this specific case, we could remove those lines completely. no need to define empty InfoLabels :)

File: addons/skin.confluence/720p/MyPVRRecordings.xml [hunk 2]
```diff
@@ -296,28 +294,23 @@
 					<left>10</left>
 					<top>548</top>
 					<width>290</width>
-					<height>12</height>>
-					<info>PVR.backenddiskspaceprogr</info>
-					<visible>!IntegerGreaterThan(PVR.backenddiskspaceprogr,100)</visible>
-				</control>
+					<height>12</height>&gt;
```
Comment (BigNoid) [inline addons/skin.confluence/720p/MyPVRRecordings.xml]: Double closing ">" here (was already there)

Comment (phil65) [inline addons/skin.confluence/720p/MyPVRRecordings.xml]: nice, thanks. :) (wondering a bit why the XMLParser didnt throw any mistake / warning)

File: addons/skin.confluence/720p/includes.xml [hunk 3]
```diff
@@ -1,20 +1,19 @@
...
+	<include file="ViewsAddonBrowser.xml" />
+	<include file="ViewsLiveTV.xml" />
+	<include file="ViewsPVRGuide.xml" />
+	<include file="ViewsWeather.xml" />
+	<include file="IncludesCodecFlagging.xml" />
+	<include file="IncludesHomeRecentlyAdded.xml" />
+	<include file="IncludesHomeMenuItems.xml" />
+	<include file="IncludesPVR.xml" />
+	<include file="IncludesBackgroundBuilding.xml" />
 	<constant name="FanartCrossfadeTime">500</constant>
```
Comment (BigNoid) [inline addons/skin.confluence/720p/includes.xml]: These need converting too

Comment (phil65) [inline addons/skin.confluence/720p/includes.xml]: hmm can I safely assume that constants are always used for animations? Or do i have to do them manually?

**Comment by phil65** [comment]
- also scaled constants and removed that double ">"

**Comment by phil65** [comment]
no more scaling constants, as told in IRC. :)

**Comment by ronie** [comment]
very nice! and agreed with everything you said above.

two things:
- the rounding issues, most new values are ok as far as i'm concerned,
only the few occasions where the new value is 188 or 262 should be changed to 200 and 250 imo
- the cosmetic changes, you're placing the comments below the actual code.
this is not preferable, in programming languages comments are usually above the actual code or besides it.

all in all a welcome change, i'm glad you took the time and made the effort to change this issue!

**Comment by phil65** [comment]
@ronie, I have no influence how the XMLWriter deals with comments so I would have to "fix" that manually. Is that worth it?
Will probably also change the few 188ers etc by hand hand then. Any comment on the values I am changing? Did I get all correctly?

**Comment by ronie** [comment]
XMLWriter probably just sees it as two elements on one line and decides to split them.
i don't mind doing some manual monkey work to fix those up once this has been merged.

i don't think you should change the 'autoscroll' and 'scrolltime' elements?
a quick test tells me that those are not affected by the effectslowdown value.

**Comment by phil65** [comment]
Ok, that would mean that only "time" and "delay" attributes for "animation" and "effect" elements are affected by effectslowdown, right?

**Comment by ronie** [comment]
yup. i've searched the code and it's the only reference to effectslowdown i can find.

https://github.com/xbmc/xbmc/blob/master/xbmc/guilib/VisibleEffect.cpp#L41

**Comment by phil65** [comment]
@ronie ok, updated PR!

**Comment by ronie** [comment]
thx!

**Pull request merged**